var/spack/repos/builtin/packages/umoci/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Umoci(MakefilePackage):
```

Comment:
Won't the build phase crash without a Makefile?

Reply:
Sorry for my fumbling around here - I think I finally got it right.  The new build phase creates the directory infrastructure and untars the src, which contains the Makefile.  Re: your earlier questions about leaving `expand=False` I started going down the path of editing the Makefile and setting the `PROJECT = .` (go apparently does not like FQPN's).  This led to a nightmare of fetching additional go packages off of github and placing them in proper subdirs using spack resources.  I eventually hit a roadblock when some subpackage required umoci to also live in one of these subdirs, so I ended up just giving in ...